File: app/src/main/java/no/nav/tilbakemeldingsmottak/rest/common/epost/HtmlContent.java
package no.nav.tilbakemeldingsmottak.rest.common.epost;


public class HtmlContent {

    private String contentString = "";

    public String getContentString() {
        return contentString;
    }

    public void addParagraph(String fieldname, String content) {
        String paragraph = createParagraph(fieldname, content);
        this.contentString += paragraph;
    }

    private String createParagraph(String fieldname, String content) {
        return String.format("<p><b>%s:</b> %s</p>", fieldname, content);
    }
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/rest/common/epost/HtmlContent.kt
package no.nav.tilbakemeldingsmottak.rest.common.epost

class HtmlContent {
    var contentString = ""
        private set

    fun addParagraph(fieldname: String, content: String) {
        val paragraph = createParagraph(fieldname, content)
        contentString += paragraph
    }

    private fun createParagraph(fieldname: String, content: String): String {
        return String.format("<p><b>%s:</b> %s</p>", fieldname, content)
    }
}
